docs/howto/system_testing.md

```diff
@@ -637,6 +643,9 @@ agent:
       cd /opt/oracle/instantclient_21_4
 ```
 
+**IMPORTANT**: The provisioning script must exit with a code different from zero in case any of the commands defined fails.
+That will ensure that the docker build step run by `elastic-package` fails too.
```

Comment:
Add a note about the importance of ensuring that the provisioning script fails if any command of the script fails.